packages/components/forms/src/text-input/types.ts

```diff
@@ -0,0 +1,22 @@
+import { BaseInputInternalProps } from '../base-input/types';
```

Comment:
I should not probably do it, but we would need to refactor it anyway.

Reply:
I think it's better if you export the type in `/base-input/index.ts`
and then here you keep importing from `../base-input`

wdyt?

Reply:
ok, I can try that. Maybe that's better, although it is temporary. we would approach it differently anyway right?

Reply:
hmmm, seems like when I import:

```
export type { BaseInputProps } from './BaseInput';
export type { BaseInputInternalProps } from './types';
```
in the `BaseInput` index. the build of the form package fail. I don't really get why.

Reply:
@gui-santos @massao I am not able to make it work in a different way than importing from `'../base-input/types'`. 🤷‍♀️